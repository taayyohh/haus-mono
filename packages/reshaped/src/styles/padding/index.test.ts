import getPaddingStyles from "./index";

describe("Styles/Padding", () => {
	test("handles positive value", () => {
		expect(getPaddingStyles(4)).toMatchSnapshot();
	});

	test("handles 0 value", () => {
		expect(getPaddingStyles(0)).toMatchSnapshot();
	});

	test("handles undefined value", () => {
		expect(getPaddingStyles()).toMatchSnapshot();
	});

	test("handles responsive value", async () => {
		expect(getPaddingStyles({ s: 4, m: 0, l: 2 })).toMatchSnapshot();
	});
});
