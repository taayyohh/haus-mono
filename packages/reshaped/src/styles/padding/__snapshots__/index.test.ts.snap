// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Styles/Padding > handles 0 value 1`] = `null`;

exports[`Styles/Padding > handles positive value 1`] = `
{
  "classNames": "_root_f9ee36",
  "variables": {
    "--rs-p-s": 4,
  },
}
`;

exports[`Styles/Padding > handles responsive value 1`] = `
{
  "classNames": "_root_f9ee36",
  "variables": {
    "--rs-p-l": 2,
    "--rs-p-m": 0,
    "--rs-p-s": 4,
  },
}
`;

exports[`Styles/Padding > handles undefined value 1`] = `null`;
